### PR TITLE
Update polkadot & substrate

### PR DESCRIPTION
**Pull request by cecton**

### CODE REVIEW

File: collator/src/lib.rs [hunk 1]
```diff
@@ -111,15 +111,16 @@ where
 	fn produce_candidate(
 		&mut self,
 		_relay_chain_parent: PHash,
-		status: LocalValidationData,
+		_global_validation: GlobalValidationSchedule,
```
Comment (cecton) [inline collator/src/lib.rs:114]: Not used

File: collator/src/lib.rs [hunk 2]
```diff
@@ -479,9 +480,15 @@ mod tests {
 		let collation = collate(
 			Default::default(),
 			id,
+			GlobalValidationSchedule {
+				block_number: 0,
+				max_code_size: 0,
+				max_head_data_size: 0,
+			},
```
Comment (cecton) [inline collator/src/lib.rs:487]: This is not used anyway

File: runtime/src/validate_block/tests.rs [hunk 3]
```diff
@@ -44,8 +44,12 @@ fn call_validate_block(
...
 	let mut ext_ext = ext.ext();
 	let params = ValidationParams {
-		block_data: block_data.encode(),
-		parent_head: parent_head.encode(),
+		block_data: BlockData(block_data.encode()),
+		parent_head: HeadData(parent_head.encode()),
+		code_upgrade_allowed: None,
+		max_code_size: 1024,
+		max_head_data_size: 1024,
+		relay_chain_height: 1,
```
Comment (cecton) [inline runtime/src/validate_block/tests.rs:52]: I took the defaults from what I found in substrate

File: test/parachain/src/command.rs [hunk 4]
```diff
@@ -14,54 +14,129 @@
 // You should have received a copy of the GNU General Public License
 // along with Cumulus.  If not, see <http://www.gnu.org/licenses/>.
 
+#![allow(unused_imports)]
+
```
Comment (bkchr) [inline test/parachain/src/command.rs:18]:
```suggestion
```

File: test/parachain/src/command.rs [hunk 5]
```diff
@@ -71,74 +146,203 @@ pub fn run(version: sc_cli::VersionInfo) -> sc_cli::Result<()> {
...
+					is_dev,
+					net_config_dir,
+					client_id,
+					node_name,
+					node_key,
+				),
+				!network_params.no_private_ipv4,
+			)
+		} else {
+			unreachable!("NetworkParams is always available on RunCmd; qed");
```
[the remark this answers is not in the document]
Comment (cecton) [reply test/parachain/src/command.rs:294]: fair point

**Pull request merged**